submissions/m-ruslan/memory-pair-game/app.js

```diff
@@ -0,0 +1,135 @@
+// main logic of game
+document.addEventListener("DOMContentLoaded", () => {
+  let cardID, cardValue, prevCardId, prevCardValue;
+  let qOfOpenCards = 0;
```

Comment:
Don’t use abbreviations like this `qOfOpenCards`- it’s unclear. Why not write full names?.
card1ID, card2ID - don't use numbers in names. Just don't :) arrOfCards also a bad idea - you show data type in the variable name and in JS it’s redundant